src/dndbeyond.tsx

```diff
@@ -24,6 +25,97 @@ const DEFAULT_THEME = 'dddice-standard';
 let characterId;
 let user: IUser;
 
+let hasHookedAux = false;
+let lastHealth: number | undefined;
+let lastTempHealth: number | undefined;
+
+function getCharacterID(): string | null {
+  const characterIdMatch = window.location.pathname.match(/characters\/(.+)/);
+  if (characterIdMatch.length > 0) {
```

Comment:
```suggestion
  if (characterIdMatch?.length > 0) {
```

characterIdMatch can be null. This caused the rolls to no longer work on the encounter builder. I fixed this in main too